.github/workflows/devmode-test-build.yml

```diff
@@ -2,24 +2,27 @@ name: production build
 
 on:
   push:
-    branches: [ "master" ]
+    paths-ignore:
```

Comment:
You don't want this workflow run on pushes to master anymore?

Reply:
It will run even then, right now with that choice it would ONLY be run when pushing to master. It will not be run for any other branch (which I sort of disliked, I'd like to get the branch I push to be test-built).

Reply:
https://github.com/PelionIoT/mbed-edge/actions/runs/9425867003 --> there it was run against master.